packages/compass-components/src/hooks/use-toast.spec.tsx

```diff
@@ -67,16 +72,6 @@ describe('useToast', function () {
   });
 
   describe('with timeout', function () {
-    let clock;
-
-    beforeEach(function () {
-      clock = sinon.useFakeTimers();
```

Comment:
Sinon doesn't polyfill `performance` object correctly, causing react to blow up when running tests. Replace with a test implementation that doesn't require faking timers (but a bit slower in execution)